Conflicting declarations under `permissionutil`

Platform: `termux/andorid`
```
~ $ go install -v [github.com/projectdiscovery/httpx/cmd/httpx@latest](http://github.com/projectdiscovery/httpx/cmd/httpx@latest)
[github.com/projectdiscovery/utils/permission](http://github.com/projectdiscovery/utils/permission)
# [github.com/projectdiscovery/utils/permission](http://github.com/projectdiscovery/utils/permission)
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_other.go:10:6: checkCurrentUserRoot redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_linux.go:16:6: other declaration of checkCurrentUserRoot
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_other.go:15:6: checkCurrentUserCapNetRaw redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.58/permission/permission_linux.go:21:6: other declaration of checkCurrentUserCapNetRaw
```

--- a/permission/permission_linux.go
+++ b/permission/permission_linux.go
@@ -1,4 +1,4 @@
-//go:build linux && !(armv7l || armv8l)
+//go:build linux && !(armv7l || armv8l) && !android
 
 package permissionutil
 

